java/acorn-graphql/src/test/resources/graphql/nutshop-schema.tools.json

```diff
@@ -12,7 +12,7 @@
         },
         "customerid" : {
           "type" : "integer",
-          "description" : " The unique identifier for the customer."
+          "description" : "customerid: The unique identifier for the customer."
```

Comment:
Is this correct?

Reply:
Yes, this is an edge case that triggered a bug: if the user includes the field name in the comment it caused an issue. This is fixed now and I added it here as a test case.